tests/test_pytest_cpp.py

```diff
@@ -628,6 +628,14 @@ def test_catch2_failure(exes):
         )
         assert len(failures) == 2
 
+        # test exceptions
+        failures, _ = facade.run_test(exes.get(f"catch2_error{suffix}"), "Error")
+        assert len(failures) == 1
+
+        fail1 = failures[0]
```

Comment:
Minor, this does the same but is more compact:

```suggestion
        [fail1] = failures
```